src/cljs/athens/devcards/styling_with_stylefy.cljs

```diff
@@ -132,3 +134,20 @@
   ```clojure
   [:div (use-style cursor-trail-style) {:style {:left x :top y}}]
   ```")
+
+
+(defcard-doc "
+  Avoid creating styles that will be frequently updated, because this forces Stylefy to create a new class for each update.
+  
+  In these cases, pass the style directly to the element to update it inline.
+  
+  ```clojure
+  [:div (use-style cursor-trail-style) {:style {:left x :top y}}]
+  ```")
```

Comment:
Needs `merge`

```
(defn button-with-dynamic-position [text color x y]
  [:button (merge (use-style (custom-primary-button-style color))
                  {:style {:left x :top y}})
    text])
```
^ 
Also makes more sense to wrap in a function to emphasize parameterization. 

https://github.com/Jarzka/stylefy#combine--parametrise-styles